src/test/java/io/appium/java_client/pagefactory_tests/widget/tests/WidgetTest.java

```diff
@@ -1,20 +1,15 @@
 package io.appium.java_client.pagefactory_tests.widget.tests;
 
-import static java.util.Arrays.copyOf;
 import static org.openqa.selenium.support.PageFactory.initElements;
 
 import io.appium.java_client.pagefactory.AppiumFieldDecorator;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.openqa.selenium.WebDriver;
 
 public abstract class WidgetTest {
```

Comment:
is this class used as a parent class for `ExtendedWidgetTest`? if yes, then these classes can be merged

Reply:
Yes, still we  are using